### ISSUE TITLE
Interface with smstools

### ISSUE THREAD

**Issue by v0d0r** [playsms/docker-playsms#9]
Not a bug or issue per say. Might just be my own ignorance but how do I interface with SMS tools when running playsms from a container?  Do I need to map a tty port to the container and where do I configure smstools?  Or does smstools not run inside playsms do I somehow need to talk to the smstools binary outside of the playsms docker?

**Comment by tobsowo** [comment]
Did you get this resolved?

**Comment by v0d0r** [comment]
Yes this was sorted. My lack of docker knowledge at the time.  I do not use playsms anymore but for what its worth this is the last working docker compose file I used.  You basically just need to give your container access to the devices. In this example I had 2 modems that mounted on the docker host as ttyUSBx and then you map them to the docker guest.

```version: '3.3'

services:
  playsms:
    image: v0d0r/playsms:latest
    container_name: playsms
    restart: unless-stopped
    devices:
      - /dev/ttyUSB0:/dev/ttyUSB0
      - /dev/ttyUSB1:/dev/ttyUSB1
    ports:
      - "2222:22"
      - "8181:80"
    volumes:
      - /mnt/nfs/swarm/dockervolumes/playsms/smstools/etc/smsd.conf:/etc/smsd.conf
      - /mnt/nfs/swarm/dockervolumes/playsms/smstools/log:/var/log/smstools
      - /mnt/nfs/swarm/dockervolumes/playsms/gammu/gammu.conf:/etc/gammu-smsdrc
    environment:
      - TZ=Africa/Johannesburg```

**Comment by tobsowo** [comment]
Thank you so much for sharing.

What do you now use?

**Comment by v0d0r** [comment]
At the time I had a use case to use SMS to automate some things remotely based on incoming sms and content it received.  Since then I don't run that project anymore but if I require the use of SMS again playsms is always my goto for small scale setups with local modems. For something at scale then still playsms but integrated with a provider that can send on scale such as Clickatell or Twilio.

**Comment by tobsowo** [comment]
Alright, thanks.
